### DISCUSSION TITLE
Quartz.net with XML configuration as a web application

### DISCUSSION THREAD

**Discussion by nirmalk323** [quartznet/quartznet#972] [General]
Can we implement Quartz.net with XML configuration as a web application? Is it possible ? Any reference is there if already implemented?

**Comment by lahma** [comment]
Have you checked the [ASP.NET Core example application](https://github.com/quartznet/quartznet/tree/master/src/Quartz.Examples.AspNetCore) that also demonstrates XML file usage?

**Reply by nirmalk323** [reply]
hi @lahma , thanks for the reference. Let me go through that and come back for doubts if any.
